lib/util/pssh.js

```diff
@@ -92,16 +106,18 @@ shaka.util.Pssh = class {
     }
 
     const dataSize = box.reader.readUint32();
-    box.reader.skip(dataSize);  // Ignore the data section.
+    const data = box.reader.readBytes(dataSize);
+    const pssh = shaka.util.Pssh.createPssh(data, systemId);
```

Comment:
Why createPssh instead of getting a slice of the original data?  This loses data.  For example, if the input is a v1 pssh, this reconstructs a v0 pssh without the key IDs.

Reply:
`box.reader` includes the systemId and the data, but not the full pssh,  I'm going to rebuild it with the correct data (including keyIds) and the correct flag for version 1. Is valid for you?

Reply:
I updated the PR with the change to get the original data.